src/test-runner.sh

```diff
@@ -63,24 +79,18 @@ for test in ${tests}; do
 
     failed=0
 
-    ${test} >${workspace}/test_output 2>${workspace}/test_output_err || true
+    ${test} >${workspace}/test_output 2>${workspace}/test_output_err || test_error=1
```

Comment:
Would be nice to catch the error code and display it later instead.... 
May be, this is possible: ```... || test_error=$?```